chitterView.js

```diff
@@ -0,0 +1,61 @@
+class ChitterView {
```

Comment:
View methods are well explained and should work for using the page locally. It is missing the API interface